lib/create-contentful-api.js

```diff
@@ -198,6 +195,21 @@ export default function createContentfulApi ({
     const resolveLinks = shouldLinksResolve(query)
     return pagedSync(http, query, resolveLinks)
   }
+  /**
+  * Parse raw json data into collection of entry objects.
+  * Links will be resolved also
+  * @param {Object} data json data represting collection of entries
+  * @example
+  * <pre>
+  * let data = {};
```

Comment:
this example is misleading and would through an error, when I'd copy and paste it, as `entries` is not defined.